### PR TITLE
fix #1513 Make Restart button more visible

### LINKED ISSUE #1513: Make Restart button more visible
**Problem:**
New users can't spot the activated "Restart" button after installing a module.

**Solution:**
Should make it more noticeable. E.g.: display the button in a new modal dialog.

**Proposal of changes:**
Display the button a modal dialog, like all other confirmations.

### PR DESCRIPTION
**Pull request by anivirtoway**
fix  #1513 Make Restart button more visible by showing modal dialog for user

### CODE REVIEW

**Comment by yecli** [comment]
Could you also add video with the changes demonstration?

File: VirtoCommerce.Platform.Web/Localizations/ru.VirtoCommerce.Platform.json [hunk 1]
```diff
@@ -804,6 +804,10 @@
           "title": "Перезагрузка приложения",
           "message": "Вы действительно хотите перезапустить приложение?"
         },
+        "app-restart-required": {
+          "title": "Требуется перезагрузка",
+          "message": "Требуется перезагрузка приложения, перегрузить сейчасц?"
```
Comment (yecli) [inline VirtoCommerce.Platform.Web/Localizations/ru.VirtoCommerce.Platform.json]: Redundant "ц" in "сейчасц".

**Comment by niksan321** [comment]
restart_dialog

**Review by yecli — CHANGES_REQUESTED** [review]

**Review by yecli — APPROVED** [review]

**Comment by yecli** [comment]
Created under wrong account

**Pull request closed without merging**